partitioned-pipe-two-phase/monolithic/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e -u
+
+blockMesh
+cp -r 0.orig 0
+setFields
+touch monolithic.foam
+
+../../../tools/run-openfoam.sh "$@"
+. ../../../tools/openfoam-remove-empty-dirs.sh && openfoam_remove_empty_dirs
```

Comment:
There are too many levels here (check also other files):

```suggestion
../../tools/run-openfoam.sh "$@"
. ../../tools/openfoam-remove-empty-dirs.sh && openfoam_remove_empty_dirs
```

Reply:
Implementing directly. No need to rerun anything.